### ISSUE TITLE
Locked out from webgui :(

### ISSUE THREAD

**Issue by Lindloven** [wuseman/TG799VAC-XTREME-17.2-MINT#8]
I rooted my router and installed the custom webgui. Then I changed root password and some other stuff I can't remember.
After that, connecting to the router gave me "403 forbidden nginx"
I then did a Physical factory reset and was able to open the Telia login-screen but when I entered the factory password and hit enter I was locked out again...
Is there anything I can do?

**Comment by kevdagoat** [comment]
Post the command `logread -e nginx` and reload the web interface or login

Sent from Mail for Windows 10

From: Lindloven
Sent: Wednesday, 17 July 2019 4:19 AM
To: wuseman/TG799VAC-XTREME-17.2-MINT
Cc: Subscribed
Subject: [wuseman/TG799VAC-XTREME-17.2-MINT] Locked out from webgui :( (#8)

I rooted my router and installed the custom webgui. Then I changed root password and some other stuff I can't remember.
After that, connecting to the router gave me "403 forbidden nginx"
I then did a Physical factory reset and was able to open the Telia login-screen but when I entered the factory password and hit enter I was locked out again...
Is there anything I can do?
